.gitignore

```diff
@@ -1,2 +1,2 @@
+/lib
 /node_modules
-/.vscode
```

Comment:
Would entries like this not be better suited to the developer's `~/.gitignore`?

Reply:
I don't see any reason it can't be in both, this also says explicitly that we don't want .vscode files committed to the repo.  That said it's not the end of the world to remove it but I don't see any benefit to doing so

Reply:
My thinking was mostly "where does it stop? Do we add `.swp`? `.idea`? etc" but I'm not that bothered, I just removed it seems as I was in the file. I'll stick it back in.